Corona-Warn-App/src/main/java/de/rki/coronawarnapp/covidcertificate/recovery/core/storage/RecoveryCertificateContainer.kt

```diff
@@ -40,9 +40,9 @@ data class RecoveryCertificateContainer(
     }
 
     override val containerId: RecoveryCertificateContainerId
-        get() = RecoveryCertificateContainerId(certificateId)
+        get() = RecoveryCertificateContainerId(qrcodeHash)
 
-    val certificateId: String
+    val qrcodeHash: String
         get() = data.recoveryCertificateQrCode.toSHA256()
 
```

Comment:
Can you use get() = containerId.identifier here? Or remove this attribute all together ? so we have the hash logic only once

Reply:
Maybe call it qrCodeHash everywhere to avoid further confusion

Reply:
Deleted it totally